ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/GAnalytics.cpp

```diff
@@ -78,7 +81,12 @@ void GAnalytics::init()
 #endif // CPP_VIEWER / QML_VIEWER
 
   const auto os = QOperatingSystemVersion::current();
-  m_defaultParameters.insert("operating_system", QString{"%1 %2.%3"}.arg(os.name(), os.majorVersion(), os.minorVersion()));
+  m_defaultParameters.insert("operating_system", QString{"%1 %2.%3"}.arg(os.name()).arg(os.majorVersion()).arg(os.minorVersion()));
```

Comment:
This is an [anti-pattern](https://github.com/KDE/clazy/blob/master/docs/checks/README-qstring-arg.md). 

Something that jumps out to me is `os.majorVersion()` and `os.minorVersion()`. Are those ints? In that case, you may want this instead:
```suggestion
  m_defaultParameters.insert("operating_system", QString{"%1 %2.%3"}.arg(os.name(), QString::number(os.majorVersion()), QString::number(os.minorVersion())));
```

(warning - syntax not validated - be sure to test it)